zetaclient/chains/solana/rpc/rpc.go

```diff
@@ -122,6 +127,25 @@ func GetSignaturesForAddressUntil(
 	return allSignatures, nil
 }
 
+// GetTransactionWithMaxVersion fetches a transaction with the given signature and max version.
+func GetTransactionWithMaxVersion(
```

Comment:
Let's hide the complexity of specifying max tx version as this rpc package is used only in zetaclient
```go
var ErrUnsupportedTxVersion = errors.New("unsupported tx version")

// GetTransaction fetches tx by the signature. Might also return ErrUnsupportedTxVersion
func GetTransaction(ctx context.Context, c interfaces.SolanaRPCClient, sig solana.Signature) (*rpc.GetTransactionResult, error) {

 // ...
}

```

Reply:
The version is used by unit test to verify different behaviors [here](https://github.com/zeta-chain/node/blob/fix-solana-unsupported-tx-version-0/zetaclient/chains/solana/rpc/rpc_live_test.go#L51)